scripts/template_upload_all.py

```diff
@@ -255,7 +256,7 @@ def browse_directory(dir_url):
         logger.exception("Skipping: %r", dir_url)
         return None
 
-    rhevm_pattern = re.compile(r'<a href="?\'?([^"\']*(?:rhevm\.ova|ovirt)[^"\'>]*)')
+    rhevm_pattern = re.compile(r'<a href="?\'?([^"\']*rhevm[^"\'>]*)')
```

Comment:
Let's use an explicit regex here to get the qcow2 image, and not rely on the order in which these are matched.

L 275 below just picks the first image name in the list, which _could_ be the ova image instead.